src/python/pants/backend/python/goals/setup_py.py

```diff
@@ -322,6 +322,21 @@ class SetupPyGeneration(Subsystem):
     @classmethod
     def register_options(cls, register):
         super().register_options(register)
+        # Generating setup is the more aggressive thing to do, so we'd prefer that the default
+        # be False. However that would break widespread existing usage, so we'll make that
+        # change in a future deprecation cycle.
```

Comment:
Do we prefer manual setup.pys? Iiuc, that means the user would need to reason about inter-project dependencies, giving them more manual work that Pants could automate.

Reply:
What if instead we always default to generating, and have ./pants tailor detect the setup.py files and set generate=False?

Imo, the default should reflect our desired end state after Pants adoption. Leverage ./pants tailor to handle the starting point. 

I'm not super convinced a subsystem option is necessary here. Options fatigue

Reply:
I don't think we know enough to say that this is the desired end state. The dependency and code ownership aspects of setup generation are complex and hard to understand.

Reply:
And kinda the whole point here is that we don't know what to detect - you could be using any PEP-517 build backend...